deploy/koku-script-runner-job.yaml

```
@@ -0,0 +1,9 @@
+---
+apiVersion: cloud.redhat.com/v1alpha1
+kind: ClowdJobInvocation
+metadata:
+  name: koku-script-runner-job
```

2 nitpicks:
```suggestion
  name: koku-cji
```
Let's keep this name generic because we may want to add other jobs to the jobs list below. Also, can we change this filename to `koku-cji.yaml`?